First part of the shellcheck linter fixes

drybjed opened the pull request:
This patch fixes some of the issues in shell scripts included in DebOps, found by the 'shellcheck' script.

drybjed commented on debops/__init__.py:
```diff
@@ -169,7 +169,7 @@ def padlock_unlock(encrypted_path):
 
     if (not os.path.exists(keyfile) or
-        not os.path.exists(os.path.join(encrypted_path, crypted_configfile))):
+            not os.path.exists(crypted_configfile)):
```
I think that the identation is wrong here? Unless that's how Python line continuation works?

johbo replied:
yes and no: In if conditions the indentation of multiple lines is often done this way so that it is obvious where the code inside of the if condition starts. I've just noticed it actually because my spacemacs was complaining to me :D

There is some theory behind it from the pep8 standard: https://www.python.org/dev/peps/pep-0008/#indentation

Not sure if you follow it or not. I can change the indentation back if you prefer that style.

drybjed replied:
I guess it works, it can stay tnen. DebOps scripts aren't PEP-8 ready, maybe they will be in the future.

drybjed committed:
[debops.pki] Fix issues reported by shellcheck

drybjed committed:
[debops.dhparam] Fix shellcheck issues

drybjed committed:
[debops.core] Fix shellcheck issues

drybjed committed:
[debops.dokuwiki] Fix shellcheck issues

drybjed committed:
[debops.debops] Fix shellcheck issues

drybjed committed:
[debops.bootstrap] Fix shellcheck issues

drybjed committed:
[debops.nginx] Fix shellcheck issues

drybjed committed:
[debops.mailman] Fix shellcheck issues

drybjed committed:
[debops.proc_hidepid] Fix shellcheck issues

drybjed committed:
[debops.php] Fix shellcheck issues

drybjed committed:
[debops.gitusers] Fix shellcheck issues

drybjed committed:
[debops.slapd] Fix shellcheck in slapd-config

This patch fixes some of the issues found by the 'shellcheck' linter in
'slapd-config' script. Some of the reported issues are ignored, since
this is an external script; they might be fixed or updated later when
the correct solution is developed.

drybjed committed:
[debops.slapd] Fix shellcheck issues

drybjed committed:
[debops.sshd] Fix shellcheck issues
